Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -768,11 +756,10 @@ public Task<object> DeserializeJson(Type type, Stream stream)
 
         public Task ProcessWebSocketRequest(HttpContext context)
         {
-            // TODO
-            return ((WebSocketSharpListener)_socketListener).ProcessWebSocketRequest(context);
+            return _socketListener.ProcessWebSocketRequest(context);
         }
-        //TODO Add Jellyfin Route Path Normalizer
 
+        //TODO Add Jellyfin Route Path Normalizer
```

Comment:
pls educate me, what's a "route path normalizer"?

Reply:
It just prepends to defined request routes such that /whatever also matches /emby/whatever